components/tools/OmeroWeb/omeroweb/webclient/webclient_gateway.py

```diff
@@ -179,6 +180,12 @@ def changeActiveGroup(self, gid):  # TODO: should be moved to ISession
             return False
 
     def getOrphanedContainerSettings(self):
+        """
+        ** Deprecated ** Use :meth:`BlitzGateway.getOrphanedContainerSettings`.
+        """
+        warnings.warn(
+            "Deprecated. Use BlitzGateway.getOrphanedContainerSettings()",
```

Comment:
Here (and in the line above) you need to use BlitzGateway.getOrphanedImagesSettings()  (renamed)